.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: Unit test affected projects
-        run: npx nx affected -t unit-test --parallel=3 --coverage
+        run: npx nx affected -t unit-test --parallel=3 --coverage.enabled --coverage.reporter=text
```

Comment:
coverage reports are now specified over the CLI only?

Reply:
The Nx test targets don't collect coverage by default. This was already the case before, the change was to use `--coverage.enabled` because `--coverage` is broken.

Reply:
I was just wondering as I guess I saw it in both places but made a comment there.